### PR TITLE
fix(additional-ips.order): in public IP creation display only v4 ip addresses in step 3

### PR DESCRIPTION
**Pull request by brofik**
ref: MANAGER-11506

| Question         | Answer
| ---------------- | ---
| Branch?          | `master` 
| Bug fix?         | yes
| New feature?     | no
| Breaking change? | no
| Tickets          | Fix #MANAGER-11506 
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] Breaking change is mentioned in relevant commits

## Related

Re-opend https://github.com/ovh/manager/pull/9366 that was closed automatically because of `stale` state.

### CODE REVIEW

**Review by JacquesLarique — APPROVED** [review]

**Review by sachinrameshn — APPROVED** [review]

File: packages/manager/modules/pci/src/projects/project/additional-ips/order/order.controller.js [hunk 1]
```diff
@@ -236,10 +237,20 @@ export default class AdditionalIpController {
     return this.createFloatingIp();
   }
 
+  static checkInstanceWithIpAddressV4(instance) {
+    return filter(
+      instance.ipAddresses,
+      (network) => network.type === 'private' && network.version === 4,
+    )?.length;
```
[the remark this answers is not in the document]
Comment (jmalnuit) [reply packages/manager/modules/pci/src/projects/project/additional-ips/order/order.controller.js:244]: @brofik Miss to take in account this feedback before prod this PR :)

[the remark this answers is not in the document]
Comment (jmalnuit) [reply packages/manager/modules/pci/src/projects/project/additional-ips/order/order.controller.js:244]: @brofik Miss to take in account this feedback before prod this PR :)

File: packages/manager/modules/pci/src/projects/project/additional-ips/order/order.controller.js [hunk 2]
```diff
@@ -208,8 +208,9 @@ export default class AdditionalIpController {
   instanceChange(instance) {
     this.privateNetworks = filter(
       instance.ipAddresses,
-      (network) => network.type === 'private',
+      (network) => network.type === 'private' && network.version === 4,
```
Comment (sachinrameshn) [inline packages/manager/modules/pci/src/projects/project/additional-ips/order/order.controller.js:211]: Can you please take version from constant ?

**Review by sachinrameshn — CHANGES_REQUESTED** [review]

**Review by anooparveti — DISMISSED** [review]

File: packages/manager/modules/pci/src/projects/project/additional-ips/order/order.controller.js [hunk 3]
```diff
@@ -236,10 +237,19 @@ export default class AdditionalIpController {
     return this.createFloatingIp();
   }
 
+  static checkInstanceWithIpAddressV4(instance) {
+    return instance?.ipAddresses.filter(
+      (network) => network.type === 'private' && network.version === 4,
```
Comment (sachinrameshn) [inline packages/manager/modules/pci/src/projects/project/additional-ips/order/order.controller.js:242]: same here as well.

**Review by JacquesLarique — DISMISSED** [review]

**Review by anooparveti — APPROVED** [review]

**Review by darsene — APPROVED** [review]

**Pull request closed without merging**